entities/src/main/java/com/redvelvet/entities/error/ErrorType.kt

```diff
@@ -1,12 +1,7 @@
 package com.redvelvet.entities.error
 
-sealed class ErrorType : Exception() {
-    data object NullData : ErrorType()
-    data object Network : ErrorType()
-    data object Server : ErrorType()
-    data object BadRequest : ErrorType()
-    data object UnAuthorized : ErrorType()
-    data object InvalidUsernameOrPassword : ErrorType()
-    data object EmailNotVerified : ErrorType()
-    data object InvalidInput : ErrorType()
-}
\ No newline at end of file
+open class MovieError: Exception()
+class NullResultError: MovieError()
+class NetworkError: MovieError()
+class ServerError: MovieError()
+open class ValidationError: MovieError()
```

Comment:
should make class for invaild user name and password and extend it from validation class